test/views/html/cards/LatestNewsAndUpdatesViewSpec.scala

```diff
@@ -44,20 +44,5 @@ class LatestNewsAndUpdatesViewSpec extends ViewSpec {
         Messages("label.latest_news_and_updates")
       )
     }
-
```

Comment:
Keep the tests. Just override the configValues in the localGuiceApplicationBuilder